src/main/java/net/dv8tion/jda/api/interactions/commands/OptionMapping.java

```diff
@@ -124,6 +124,18 @@ public long getAsLong()
                 return data.getLong("value");
         }
     }
+    
+    public double getAsDouble()
+    {
+        switch (type)
+        {
+            default:
+                throw new IllegalStateException("Cannot convert option of type " + type + " to double");
+            case STRING:
+            case NUMBER:
```

Comment:
Can an integer not be converted to a number?

Reply:
Added this.